Google.PowerShell.IntegrationTests/Container/Container.GkeCluster.Tests.ps1

```diff
@@ -246,27 +246,6 @@ Describe "Add-GkeCluster" {
     $clusterOneJob = Start-Job -ScriptBlock $clusterCreationWithoutUsingNodeConfigScriptBlock `
                                -ArgumentList @($gcloudCmdletsPath, $clusterOneParameter)
 
-    # Cluster Two creation.
-
-    # Create a network and extract out subnet that corresponds to region "us-central1".
-    $script:networkName = "test-network-$r"
-    $region = "us-central1"
-    $script:subnetName = New-NetworkAndSubnetwork $networkName $region
-
-    $script:clusterTwoName = "gcp-new-gkecluster-2-$r"
-    $script:clusterTwoZone = "us-central1-a"
-    $clusterTwoAdditionalZone = "us-central1-c"
-    $clusterTwoMachineType = "n1-standard-4"
-    $clusterTwoParameter = @{"clusterName" = $clusterTwoName;
-                             "zone" = $clusterTwoZone;
-                             "network" = $networkName;
-                             "subnetwork" = $subnetName;
-                             "additionalZone" = $clusterTwoAdditionalZone;
-                             "initialNodeCount" = 2;
-                             "machineType" = $clusterTwoMachineType }
-
-    $clusterTwoJob = Start-Job -ScriptBlock $clusterCreationWithoutUsingNodeConfigScriptBlock `
-                               -ArgumentList @($gcloudCmdletsPath, $clusterTwoParameter)
 
     # Cluster Three creation.
     $script:clusterThreeName = "gcp-new-gkecluster-3-$r"
```

Comment:
You should probably change some variable names so you still have sequential names (replace Three with Two).